admin/resources/js/components/poolCandidate/CreatePoolCandidate.tsx

```diff
@@ -95,11 +101,11 @@ export const CreatePoolCandidateForm: React.FunctionComponent<CreatePoolCandidat
     const onSubmit: SubmitHandler<FormValues> = async (data: FormValues) => {
       await handleCreatePoolCandidate(formValuesToSubmitData(data))
         .then(() => {
-          // TODO: Navigate to pool candidate dashboard.
+          navigate(poolTable(locale));
```

Comment:
Once a Pool Candidate is created, this should probably navigate to the `/pools/:poolId/pool-candidates`.